ecl/hqlcpp/hqlsource.cpp

```diff
@@ -3333,7 +3333,7 @@ void ChildAggregateBuilder::buildTransform(IHqlExpression * expr)
     BuildCtx transformCtx(instance->startctx);
     transformCtx.addQuotedCompound("virtual void processRows(ARowBuilder & crSelf)");
     translator.ensureRowAllocated(transformCtx, "crSelf");
-    buildTransformBody(transformCtx, expr, false, false);
+    buildTransformBody(transformCtx, expr, false, false, false);
```

Comment:
yes.  If you look at the prototype of the function that is being generated there is no parameter for the current input row - so it should not be bound.  Same goes for example below.